### DISCUSSION TITLE
Autoplay Video in Readme.md on github

### DISCUSSION THREAD

**Discussion by asnatschko** [neuefische/questions#267] [Web]
Hi all :) 
How is it possible to make an autoplay-presentation of our app-demonstration in a readme-file? 

Here the working example:
https://github.com/mar-zim/capstone-project 

Thank you in advance!

**Comment by NikRoe** [comment]
Hi Alex,

I did some research and one possible solution would be converting the video to a gif and embedding it afterwards.
I found [this article](https://stackoverflow.com/questions/4279611/how-to-embed-a-video-into-github-readme-md) on stackoverflow for further information regarding this.

This [answer](https://stackoverflow.com/questions/4279611/how-to-embed-a-video-into-github-readme-md/29842302#29842302) details it even further.

Hope this helps :)

**Reply by NikRoe** [reply]
Hi Alex,
did this solve the issue? :)

**Reply by asnatschko** [reply]
Hi Niklas! Thank you! I try it :)
